kytos/utils/users.py

```diff
@@ -0,0 +1,144 @@
+"""Module used to handle Users in Napps Server."""
+import logging
+import re
+from getpass import getpass
+
+from kytos.utils.client import UsersClient
+from kytos.utils.config import KytosConfig
+
+log = logging.getLogger(__name__)
+
```

Comment:
hum... my mistake, I haven't noticed that there is no comma between the first part of the description and the second one. Ignore this comment.